### PR TITLE
[TF-519] Set up infrastructure to connect to TPU on top of eager execution

### PR DESCRIPTION
**Pull request by burmako**
With this PR, I've been able to run the following code on TPU:

```swift
import TensorFlow
withDevice(.tpu, 0) {
  let x = Tensor<Float>([1, 2, 3, 4, 5])
  print(x + x)
}
```

This required setting the `SWIFT_TENSORFLOW_SERVER_ADDRESS` environment variable to something that can communicate with a TPU machine. Details of configuring that are currently Google3-specific, so I won't be posting them here. In TF-520, we'll be figuring out how to arrange this in a manner compatible with public CI.

### CODE REVIEW

File: stdlib/public/TensorFlow/CompilerRuntime.swift [hunk 1]
```diff
@@ -1071,7 +1099,12 @@ func _TFCEagerExecute(_ op: CTFEOp,
                       _ status: CTFStatus) {
   if _RuntimeConfig.printsDebugLog {
     debugLog("Calling _TFCEagerExecute() over: ")
-    TFE_OpPrintDebugString(op)
+    if let value = getenv("TF_CPP_MIN_LOG_LEVEL"),
```
Comment (bgogul) [inline stdlib/public/TensorFlow/CompilerRuntime.swift]: This already gets printed only if SWIFT_TENSORFLOW_ENABLE_DEBUG_LOGGING is set? Do you also want to control it with TF_CPP_MIN_LOG_LEVEL?

**Review by bgogul — APPROVED** [review]
Looks good. Left a question about TF_CPP_MIN_LOG_LEVEL

**Comment by bgogul** [comment]
@swift-ci please test tensorflow

**Comment by bgogul** [comment]
@swift-ci please clean test tensorflow

File: stdlib/public/TensorFlow/CompilerRuntime.swift (hunk 1 above)
Comment (burmako) [reply stdlib/public/TensorFlow/CompilerRuntime.swift]: If `SWIFT_TENSORFLOW_ENABLE_DEBUG_LOGGING` is set to `true` but `TF_CPP_MIN_LOG_LEVEL` is not set to `0`, compiler runtime prints `Calling _TFCEagerExecute() over: ` and then nothing. I figured that this behavior is undesirable hence this change that suggests that the user specifies `TF_CPP_MIN_LOG_LEVEL=0`.

File: stdlib/public/TensorFlow/CompilerRuntime.swift [hunk 2]
```diff
@@ -1071,7 +1099,12 @@ func _TFCEagerExecute(_ op: CTFEOp,
...
+      String(cString: value) == "0" {
+        TFE_OpPrintDebugString(op)
```
Comment (rxwei) [inline stdlib/public/TensorFlow/CompilerRuntime.swift]: Random nit on indentation of the body of multiple-line `if`:
```suggestion
      TFE_OpPrintDebugString(op)
```

Comment (rxwei) [reply stdlib/public/TensorFlow/CompilerRuntime.swift]: No need to fix in this PR since you've got CI started :)

**Comment by burmako** [comment]
Merging this PR because the only change since the recent CI success was about formatting.

**Pull request merged**